scripts/lib/zulip_tools.py

```diff
@@ -410,3 +411,18 @@ def assert_running_as_root(strip_lib_from_paths: bool=False) -> None:
     if not is_root():
         print("{} must be run as root.".format(script_name))
         sys.exit(1)
+
+def get_config_file() -> configparser.RawConfigParser:
+    config_file = configparser.RawConfigParser()
+    config_file.read("/etc/zulip/zulip.conf")
+
+    return config_file
+
+def get_deploy_options(config_file):
+    # type: (configparser.RawConfigParser) -> List[str]
+    try:
+        deploy_options = config_file.get('deployment', 'deploy_options').strip().split()
+    except (configparser.NoSectionError, configparser.NoOptionError):
+        deploy_options = []
```

Comment:
I think if we're extracting a function, we should use the approach we use in `zproject/settings.py` of having a common function that handles the `configparser` exceptions has a nice interface of the form

`get_config('deployment', 'deploy_options', [])`

Reply:
(It's possible we still want the `get_deploy_options` wrapper to avoid code duplication, but that way we have a nice generic `get_config` available in `zulip_tools.py`, which we can call for e.g. `remote_url` above as well)

Reply:
@timabbott I've made the requested changes.